### PR TITLE
Adds grunt/sass example

### PR DESCRIPTION
**Pull request by Ralt**

### CODE REVIEW

File: reference/configuration-files.rst [hunk 1]
```diff
@@ -231,6 +231,31 @@ After a Git push, you can see the results of the deployment hooks in the ``/var/
...
+
+  hooks:
+    build: |
+      cd public/profiles/project_name/themes/custom/theme_name
+      npm install
+      grunt
+
+This requires the ``package.json`` and ``Gruntfile.js`` files to be correctly setup in the theme folder.
+
+.. note::
```
Comment (GuGuss) [inline reference/configuration-files.rst]: No need for this line. It's self explicit as a YAML format.

File: reference/configuration-files.rst [hunk 2]
```diff
@@ -231,6 +231,31 @@ After a Git push, you can see the results of the deployment hooks in the ``/var/
 
 ----
 
+A good example combining dependencies and hooks: a theme compiling sass files using grunt. Here is an example of how to set this up:
+
+In the ``.platform.app.yaml`` file:
```
Comment (GuGuss) [inline reference/configuration-files.rst]: the ==> your

File: reference/configuration-files.rst (hunk 1 above)
Comment (Ralt) [inline reference/configuration-files.rst]: Fair enough

File: reference/configuration-files.rst [hunk 3]
```diff
@@ -231,6 +231,31 @@ After a Git push, you can see the results of the deployment hooks in the ``/var/
 
 ----
```
Comment (GuGuss) [inline reference/configuration-files.rst]: Remove this separator.

File: reference/configuration-files.rst [hunk 4]
```diff
@@ -231,6 +231,31 @@ After a Git push, you can see the results of the deployment hooks in the ``/var/
...
 
+A good example combining dependencies and hooks: a theme compiling sass files using grunt. Here is an example of how to set this up:
```
Comment (GuGuss) [inline reference/configuration-files.rst]: As a good example combining dependencies and hooks, you can compile your SASS files using Grunt.

**Pull request merged**